brianlong/www.validators.app
# 181897977 db copy feature

#### What's this PR do?
- Add ability to copy PingThings from production to stage

#### How should this be manually tested?
- configure database.yml to support two dbs.
- create ping thing and check if it's created in both db's

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/181897977)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
